Sources_v3/StreamChatUI/ChatChannel/ChatMessageImageGallery.swift

```diff
@@ -0,0 +1,220 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Nuke
+import StreamChat
+import UIKit
+
+open class ChatMessageImageGallery<ExtraData: UIExtraDataTypes>: View, UIConfigProvider {
+    public var didTapOnAttachment: ((_ChatMessageAttachment<ExtraData>) -> Void)? {
+        didSet { updateContent() }
```

Comment:
Why do we need to update content when someone sets a callback?

Reply:
fixed in 1dba63eb818a7079287baa574257cbce4351a8e9